spec/javascripts/models/story_spec.js

```diff
@@ -474,4 +474,16 @@ describe('Story', function() {
     });
   });
 
+  describe('changeState', function() {
+    it('sets the owner to the current user when story is started', function() {
+      this.story.set({ onwed_by_id: 123 });
+      this.story.changeState(this.story, 'started');
+      expect(this.story.get('owned_by_id')).toBe(999);
```

Comment:
At the beginning of the file has [these mocks](https://github.com/iagodahlem/cm42-central/blob/b50f30544663a36871d1a6dcae8f90798af39794/spec/javascripts/models/story_spec.js#L10).